WcaOnRails/.rubocop.yml

```diff
@@ -103,6 +103,9 @@ Performance/RedundantMatch:
 Style/NegatedIf:
   Enabled: false
 
+Style/AccessModifierDeclarations:
+  Enabled: false
```

Comment:
You can find the documentation [here](https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_style.md#styleaccessmodifierdeclarations). Personally I don't think we need this check, in most cases we use inline `private` keyword, but we have reasonable usages of the block version.